plugins/inputs/suricata/suricata.go

```diff
@@ -150,27 +157,36 @@ func flexFlatten(outmap map[string]interface{}, field string, v interface{}, del
 		}
 	case float64:
 		outmap[field] = v.(float64)
+	case string:
+		outmap[field] = v.(string)
```

Comment:
Probably also a leftover from old code, but you can simplify to
```suggestion
	case float64:
		outmap[field] = t
	case string:
		outmap[field] = t
```